app/playground/Dockerfile-playground

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+MAINTAINER James Lamb <jaylamb20@gmail.com>
+
+# TODO (jaylamb20@gmail.com):
+# Implement main ingestion service that reads from S3 and writes to our DB
```

Comment:
I guess this is the playground service.. unlike what the comment says about this being an ingestion service

Reply:
oh good call, bad copy-pasting lol